src/fidesops/ops/core/config.py

```diff
@@ -38,6 +38,7 @@ class ExecutionSettings(FidesSettings):
     task_retry_count: int
     task_retry_delay: int  # In seconds
     task_retry_backoff: int
+    identity_verification_required: bool = False
```

Comment:
I am just setting a config variable for now, but this is subject to change with https://github.com/ethyca/fidesops/issues/1009.  I think we should leave this to False until all the relevant pieces are in.